packages/salesforcedx-vscode-replay-debugger/package.json

```diff
@@ -74,6 +74,21 @@
         "language": "apex"
       }
     ],
+    "languages": [
+      {
+        "id": "apexlog",
+        "aliases": ["ApexLog", "DebugLog"],
+        "extensions": [".log"],
```

Comment:
I haven't had the chance to test this out but what happens to other files with .log extension? Does this also get applied there or does the `firstLine` attribute invalidate it?

I remember (and could be wrong) that VS Code calculate the langId of a file fuzzily by combining different criteria (extension, firstline, etc).

Reply:
`firstLine` doesn't invalidate it. I can still launch from context menu, but the launch will error out because of this other regex check we put in a while back: https://github.com/forcedotcom/salesforcedx-vscode/pull/358